src/vl-multiselect.src.js

```diff
@@ -25,10 +25,14 @@ export class VlMultiSelect extends VlSelect {
     return 'VlMultiSelectReady';
   }
 
+  constructor() {
+    super();
+    this.setAttribute('data-vl-multiselect', '');
```

Comment:
Het name attribuut en de class mag wel in de connectedCallback geset worden? Wat is het verschil/probleem hiermee dan juist?

Reply:
De vl-ui-select gaat bij het wijzigen van de succes/error attributen wachten op een dress, maar dat doet hij op basis van het data-vl-multiselect attribuut. Als dit attribuut pas gezet wordt in de connectedCallback, denkt de vl-ui-select dat het element niet dressed is, en worden de classen niet op de gegenereerde containers gezet waardoor de stijl niet goed doorkomt.

Reply:
Gotcha, is het dan ook niet defensiever als we de rest van de connectedCallback ook in de constructor zetten, of levert dat dan weer andere problemen op?

Reply:
Als ik me niet vergis, gaf dat vroeger problemen bij de dress functie, maar ik heb het nu niet opnieuw geprobeerd om geen bugs te introduceren die nog niet gedekt zijn door testen.

Reply:
Ah ok. Als dat vroeger meer problemen gaf dan moeten we dat niet perse opnieuw proberen nu :)